### PR TITLE
Integrate OneAuthTestApp build step into daily and release pipelines.

### PR DESCRIPTION
**Pull request by fadidurah**
Added oneauth test app build and download step to dailr and monthly piplines

Validation
Daily: https://identitydivision.visualstudio.com/Engineering/_build/results?buildId=1226203&view=results
Monthly: https://identitydivision.visualstudio.com/Engineering/_build/results?buildId=1226202&view=results

### CODE REVIEW

File: azure-pipelines/ui-automation/templates/build-one-auth-test-app.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    displayName: Build and Download OneAuth Test App
+    variables:
+      - name: project
+        value: OneAuth
+      - name: definitionId
+        value: 6332
+      - name: url
+        value: https://office.visualstudio.com/
+      - name: timeout
+        value: 120
```
Comment (somalaya) [inline azure-pipelines/ui-automation/templates/build-one-auth-test-app.yml:22]: we would not need timeout as 120. 60 should be ok

File: azure-pipelines/ui-automation/templates/build-one-auth-test-app.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      - name: oneAuthProjectId
+        value : 'a4d3949e-967a-4298-8b6d-53f45f6b0704'
+    timeoutInMinutes: 120
```
Comment (somalaya) [inline azure-pipelines/ui-automation/templates/build-one-auth-test-app.yml:25]: I am not sure why we have 2 timeouts. But we can cancel the build after 60 mins.

**Review by somalaya — APPROVED** [review]

File: azure-pipelines/ui-automation/templates/build-one-auth-test-app.yml (hunk 2 above)
Comment (fadidurah) [reply azure-pipelines/ui-automation/templates/build-one-auth-test-app.yml:25]: Ok i can do that

Comment (fadidurah) [reply azure-pipelines/ui-automation/templates/build-one-auth-test-app.yml:25]: We need one for the build queue, and one more for the pipeline stage

**Pull request merged**